staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -431,15 +431,33 @@ func (mounter *SafeFormatAndMount) formatAndMountSensitive(source string, target
 		}
 
 		// Disk is unformatted so format it.
-		args := []string{source}
+		args := []string{}
 		if fstype == "ext4" || fstype == "ext3" {
 			args = []string{
 				"-F",  // Force flag
 				"-m0", // Zero blocks reserved for super-user
-				source,
 			}
 		}
 
+		if mounter.MkfsNodiscard {
+			if fstype == "ext4" {
```

Comment:
Does Windows have a similar recommendation? cc @jingxu97

Reply:
@leiyiz can you follow up on if Windows has a similar recommendation? We can handle windows separately.